src/components/organisms/UiList/_internal/UiListItemButton.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <UiButton class="search-dropdown-input-row__button ui-button--text ui-list-item-button">
```

Comment:
I'm sure, that button shouldn't have block element as children <br/>
https://github.com/infermedica/component-library/pull/489#discussion_r1618696984